patches/minecraft/net/minecraft/world/biome/provider/NetherBiomeProvider.java.patch

```diff
@@ -0,0 +1,11 @@
+--- a/net/minecraft/world/biome/provider/NetherBiomeProvider.java
++++ b/net/minecraft/world/biome/provider/NetherBiomeProvider.java
+@@ -77,7 +77,7 @@
+    }
+ 
+    private static NetherBiomeProvider func_235285_d_(long p_235285_0_) {
+-      ImmutableList<Biome> immutablelist = ImmutableList.of(Biomes.field_235254_j_, Biomes.field_235252_ay_, Biomes.field_235253_az_, Biomes.field_235250_aA_, Biomes.field_235251_aB_);
++      ImmutableList<Biome> immutablelist = net.minecraftforge.common.BiomeManager.getNetherBiomes();
+       return new NetherBiomeProvider(p_235285_0_, immutablelist.stream().flatMap((p_235273_0_) -> {
```

Comment:
These should be expression lambdas and it can all be on a single line

Reply:
The lambda is not part of this PR, it is just showing due to how patches work. I believe at one point there was a ForgeFlower issue/PR related to lambdas that can be inlined which would address your comment. Regardless it doesn't actually have to do with this PR as it is't one of the lines modified.

Reply:
Oh right my bad, I misread the diff

Reply:
yeah it's a one line patch lol